### ISSUE TITLE
im not completely sure why but it seems like changing things about spawners inside of the code causes the game to crash

### ISSUE THREAD

**Issue by pYMxlobo** [Moonzel/Godot-PerfBullets#13]
but im also not completely sure about that either since it might just be spawning spawner nodes from a sub-scene might be the reason but i have tested with it inside and outside a subscene and it still crashes before godot can print any errors or anything like that. below is an unlisted video on my channel of this problem occuring.

[video](https://youtu.be/x-wee4n5IIA)

**Comment by pYMxlobo** [comment]
update: attaching a node of the same type as the planned target as a temporary starting target seems to do the trick in fixing it but it crashes even when there are 0 homing or general target node things if its not assigned so im not closing the issue

**Comment by Moonzel** [comment]
You are right, you cannot edit through code in that way because of the clashing between C++ and GDScript with GDExtension. The plugin expects to be provided with the necessary prerequisites for running, and if it does not have them, it will crash based off of the code. Please let me know if I am misunderstanding, but I am closing this as of now.
